src/OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster.cpp

```diff
@@ -34,6 +34,7 @@ Thruster::Thruster(
     : Dynamics(aName),
       satelliteSystem_(aSatelliteSystem),
       guidanceLaw_(aGuidanceLaw),
+      name_(aName),
```

Comment:
Don't think this is necessary as the `Dynamics(aName)` parent constructor should take care of that.

```suggestion
```